rules/collections/default/java.yaml

```diff
@@ -0,0 +1,6 @@
+collections:
+  - id: Collections.SparkJava
+    name: SparkJava Http Framework
+    patterns:
+      - "(?i)Spark.(post|get|delete|put).*"
```

Comment:
@sriram-privado shouldn't we also add the package name for Spark? Or it is Spark?

Reply:
`spark.Spark` is the package name. I'll add it here.